databook/physiology/ephys/np-ultra/npultra-psychedelics-accessing-data.md

```diff
@@ -0,0 +1,112 @@
+# Accessing NP Ultra & Psychedelics Data
+
+## Loading Data
+The Neuropixels Ultra & Psychedelics dataset is packaged in nwb format and can be accessed via NWBZarrIO. 
+
+Load an example session:
+```{code-cell} ipython3
+from hdmf_zarr import NWBZarrIO
+import json
+import os
+```
+```{code-cell} ipython3 
+from hdmf_zarr import NWBZarrIO 
+
+nwbfile_path_zarr = '/data/np-ultra-psychedelics/ecephys_714527_2024-05-15_13-00-23_nwb_2025-08-03_21-11-22/ecephys_714527_2024-05-15_13-00-23_experiment1_recording1.nwb'
+
+with NWBZarrIO(nwbfile_path_zarr, mode='r') as io:
+    nwbfile_zarr = io.read()
+```
+Alternatively, more than one session can be loaded into memory and stored into a list by:
```

Comment:
I've never seen this - you end up with a list of nwb files? But the zarr backend doesn't open the files, it's like a pointer to them. Does this really work?